pkgs/applications/networking/cluster/popeye/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "popeye";
+  version = "0.8.10";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "derailed";
+    repo = "popeye";
+    sha256 = "1cx39xipvvhb12nhvg7kfssnqafajni662b070ynlw8p870bj0sn";
+  };
+
+  buildFlagsArray = ''
+    -ldflags=
+      -s -w
+      -X github.com/derailed/popeye/cmd.version=${version}
+      -X github.com/derailed/popeye/cmd.commit=b7a876eafd4f7ec5683808d8d6880c41c805056a
+      -X github.com/derailed/popeye/cmd.date=2020-08-25T23:02:30Z
```

Comment:
I'm not an expert on Go packaging here, and maybe my question can be a bit amiss, but, is this info needed somehow?

What is the meaning of these `-X` flags?

Reply:
You can find the meaning of the flags here: https://golang.org/cmd/link/
But basically I use this to set the version of the package, otherwise `popeye version` will return nothing, the same is valid for `velero`.
The importpaths was extracted from here: https://github.com/derailed/popeye/blob/master/Makefile

Reply:
A small hint: if you need to use another file as an example, copy its link here. You can even click at "view file" and copy the specific line.

https://github.com/NixOS/nixpkgs/blob/142f6d8515b699e08052482a69fc4c9a0f4fd041/pkgs/applications/networking/cluster/velero/default.nix#L19

Reply:
Maybe for the future it can be done by some script. 
Again I am not an expert on it, but all the information used by those flags can be fetched by reading the git repo metadata. Maybe opening a thread on Discourse would be useful on this.

Anyway, LGTM.

Reply:
> Maybe for the future it can be done by some script.
> Again I am not an expert on it, but all the information used by those flags can be fetched by reading the git repo metadata. Maybe opening a thread on Discourse would be useful on this.
> 
> Anyway, LGTM and LGTBorg.

Good, thats what I want to to do next. Thanks for the feedback.